src/get-started/flutter-for/uikit-devs.md

```diff
@@ -2173,8 +2173,8 @@ class _SampleAppPageState extends State<SampleAppPage> {
 [layout tutorial]: {{site.url}}/ui/widgets/layout
 [`Localizations`]: {{site.api}}/flutter/widgets/Localizations-class.html
 [Material Components]: {{site.material}}/develop/flutter/
-[Material Design guidelines]: {{site.material}}/design/
-[optimized for all platforms]: {{site.material}}/design/platform-guidance/cross-platform-adaptation.html#cross-platform-guidelines
```

Comment:
this is one of those pages that doesn't have a counterpart in M3 (and won't) so I think we should keep linking to the m2 site

Reply:
[Updated ](https://github.com/flutter/website/pull/8706/commits/465fee0a6a3c77969f77223165f6d7cfb662bec5#diff-2bd4c52d8555721f7620ab1c9c3848f7f3a6cbd59d819da34d62bf3e5a96fe48)